lib/Doctrine/ORM/Tools/Console/Command/ConvertDoctrine1SchemaCommand.php

```diff
@@ -180,6 +180,7 @@ public function convertDoctrine1Schema(array $fromPaths, $destPath, $toType, $nu
                 );
             }
         }
+        unset($dirName);
```

Comment:
I know why this is applied, it just doesn't make sense if the variables are unused. I'd rather prefer extracting the loop into a private method instead.